GitCommands/RevisionReader.cs

```diff
@@ -79,6 +79,11 @@ public sealed class RevisionReader : IDisposable
                     });
         }
 
+        /// <summary>
+        /// Parents require rewrite, all commits may not be included
+        /// </summary>
+        public bool ParentsAreRewriten { get; private set; } = false;
```

Comment:
```suggestion
        public bool ParentsAreRewritten { get; private set; } = false;
```

Reply:
I re-read the xml-doc and the name over and over, and I still can't grasp the intent... 🤕

Reply:
The parents for the commits are rewritten for the filter and all commits may not be included

Reply:
This piece was dropped in this PR after a rewrite, will appear in a follow up though